package.json

```diff
@@ -125,6 +125,7 @@
     "webpack-cli": "^3.1.0"
   },
   "dependencies": {
+    "attr-accept": "^1.1.3",
```

Comment:
I wonder if we should use https://www.npmjs.com/package/accepts? seems like more used / maintained

Reply:
i chose this package to be consistent with `react-dropzone` https://github.com/react-dropzone/react-dropzone/blob/master/package.json#L84 which, according to a reviewer on jerome's pr, is commonly used across the FE org